ts/src/base/Exchange.ts

```diff
@@ -3775,13 +3776,49 @@ export default class Exchange {
         return this.safeString (market, 'symbol', symbol);
     }
 
+    handleParam (params: object, paramName: string, defaultValue = undefined) {
+        const value = this.safeValue (params, paramName, defaultValue);
+        if (value !== undefined) {
+            params = this.omit (params, paramName);
+        }
+        return [ value, params ];
+    }
+
     resolvePath (path, params) {
         return [
             this.implodeParams (path, params),
             this.omit (params, this.extractParams (path)),
         ];
     }
 
+    getListFromObjectValues (objects, key: IndexType) {
+        const newArray = this.toArray (objects);
+        const results = [];
+        for (let i = 0; i < newArray.length; i++) {
+            results.push (newArray[i][key]);
+        }
+        return results;
+    }
+
```

Comment:
`tiveStatuses = [ true, undefined ]` this syntax does not work in c#/java/etc, we can't use it

Reply:
ahh,  give me 2 min.

Reply:
sry.

Reply:
does this last commit look ok?